src/components/Sum.jsx

```diff
@@ -1 +1,3 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (<p>{`Sum of ${a} and ${b} is ${a + b}`}</p>);
```

Comment:
please do not forget about code formatting
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
);
```